app/views/components/docs/contents-list.yml

```diff
@@ -106,7 +117,7 @@ examples:
           items:
           - href: "#second-thing"
             text: No numbers here
-          - href: "#third-thing"
+          - href:
```

Comment:
Could the API omit the `href` altogether?
```
- text: 'Thing'
- text: 'Another thing'
```

Reply:
Yep, amending.